oxy/mapping.yaml

```diff
@@ -0,0 +1,54 @@
+!CANMsg
+id: 1
+desc: "accumulator status"
+fields:
+- !CANField
+    id: 1
+    name: Pack Inst Voltage
+    units: "V"
+    size: 2
+    decodings:
+    - !BigEndian
```

Comment:
I can't think of a better way to do this, but having "BigEndian" be one of the classes that gets made feels funky. I feel like BigEndian is an attribute of the way we are decoding it, not the overarching thing

Reply:
Actually potentially jk I see how you did it with the BigEndian object below and can kinda see it now